### PR TITLE
Update queries and categories

### PR DESCRIPTION
**Pull request by arnauddieumegard**

### CODE REVIEW

File: common/plugins/org.polarsys.capella.common.ui.toolkit.browser/src/org/polarsys/capella/common/ui/toolkit/browser/category/CategoryRegistry.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ private CategoryRegistry() {
     for (Entry<String, ICategory> entry : entrySet) {
       ICategory category = entry.getValue();
       // top level category and type matching
-      if (category.isTopLevel() && category.isAvailableForType(currentElement)) {
+      if (category.isTopLevel() && category.isAvailableForType(currentElement) && !category.isTechnical()) {
```
Comment (pdulth) [inline common/plugins/org.polarsys.capella.common.ui.toolkit.browser/src/org/polarsys/capella/common/ui/toolkit/browser/category/CategoryRegistry.java:75]: Can you also a new `public Set<ICategory> gatherTechnicalCategories(String viewerId, EObject currentElement)` to be able to retrieve them

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml [hunk 2]
```diff
@@ -7847,4 +7847,92 @@
...
+   <extension
+         id="x"
+         point="org.polarsys.capella.common.ui.toolkit.browser.contentProviderCategory">
+      <category
+            id="---------- FunctionalChain ----------">
+      </category>
+      <category
+            id="flatFunctions"
+            isTopLevel="true"
+            name="Flat Functions">
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:7859]: Can you name it "Involved Functions", for SystemAnalsis/Logical/Physical, and another query named "Involved Operational Activities" for OperationalAnalysis level

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml [hunk 3]
```diff
@@ -7847,4 +7847,92 @@
...
+         <categoryQuery>
+            <basicQuery
+                  class="org.polarsys.capella.core.semantic.queries.technical.queries.FunctionalChainFlatFunctions">
+            </basicQuery>
+         </categoryQuery>
+      </category>
+      <category
+            id="flatExchanges"
+            isTopLevel="true"
+            name="Flat Exchanges">
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:7875]: "Involved Exchanges" (SA/LA/PA) / "Involved Interactions" (OA)

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml [hunk 4]
```diff
@@ -7847,4 +7847,92 @@
...
+         <categoryQuery>
+            <basicQuery
+                  class="org.polarsys.capella.core.semantic.queries.technical.queries.FunctionalChainFlatExchanges">
+            </basicQuery>
+         </categoryQuery>
+      </category>
+      <category
+            id="startingFunctions"
+            isTopLevel="true"
+            name="Starting Functions">
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:7891]: "Starting Functions" (SA/LA/PA) / "Starting Operational Activities" (OA)

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml [hunk 5]
```diff
@@ -7847,4 +7847,92 @@
...
+         <categoryQuery>
+            <basicQuery
+                  class="org.polarsys.capella.core.semantic.queries.technical.queries.FunctionalChainStartingFunctions">
+            </basicQuery>
+         </categoryQuery>
+      </category>
+      <category
+            id="endingFunctions"
+            isTopLevel="true"
+            name="Ending Functions">
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:7907]: "Ending Functions" (SA/LA/PA) / "Ending Operational Activities" (OA)

File: common/plugins/org.polarsys.capella.common.ui.toolkit.browser/src/org/polarsys/capella/common/ui/toolkit/browser/category/CategoryRegistry.java (hunk 1 above)
Comment (arnauddieumegard) [reply common/plugins/org.polarsys.capella.common.ui.toolkit.browser/src/org/polarsys/capella/common/ui/toolkit/browser/category/CategoryRegistry.java:75]: Will do, is the approach good for you ?

**Review by pdulth — APPROVED** [review]

**Review by pdulth — APPROVED** [review]

**Pull request merged**